### ISSUE TITLE
Development environment setup hardcoded path

### ISSUE THREAD

**Issue by Petrox** [inventree/InvenTree#5692]
### Body of the issue

Following this:

https://docs.inventree.org/en/latest/start/docker_dev/#docker-development-server

Docs mentions ./data relative to the source will be created automatically:

"Persistent data (such as the stored database, media files, configuration files, etc) will be stored in the ./data directory (relative to the InvenTree source code directory)."

but then running

![kép](https://github.com/inventree/InvenTree/assets/3036509/ba074acc-4585-4f9a-91d5-3d0ee51b120e)

fails with:

`
petros@xubuntu20:/proj/InvenTree$ docker compose run inventree-dev-server invoke update
[+] Creating 0/0
[+] Creating 0/1ventree-inventree-dev-db-1  Recreate                        0.1s[+] Creating 0/1ventree-inventree-dev-db-1  Recreate                        0.2s[+] Creating 1/1nventree-inventree-dev-db-1  Recreate                        0.3 â Container inventree-inventree-dev-db-1  Recreated                       0.4s 
[+] Running 0/0
[+] Running 1/1nventree-inventree-dev-db-1  Starting                        0.1s â Container inventree-inventree-dev-db-1  Started                         0.2s 

Loading config file : /home/inventree/data/config.yaml
Using Python virtual environment: /home/inventree/data/env
./docker/init.sh: source: line 49: can't open '/home/inventree/data/env/bin/activate': No such file or directory
`

TLDR: following the steps described does not result in a working container because the document does not mention that the docker development environment depends on /home/inventree/data being created (and that the source must be under /home/, etc)

(I figured it out, just wanted to describe the experience with the "Quickstart guide" returning errors.)

**Comment by SchrodingersGat** [comment]
@Petrox you 100% *do not* need to have the specified `/home/inventree/` directory on your local machine - when running with docker, these directories are created *within* the docker container itself. 

The only external directory is "data" which is automatically created external to the docker container, *local* to the InvenTree source directory.

If you look at the [docker compose file](https://github.com/inventree/InvenTree/blob/master/docker-compose.yml) you can see how these directories are all mapped.

If you have seen the error above, it probably means that there is a permission issue for the docker process, to wherever you have cloned the InvenTree source code.

When you run the initial docker compose commands, do you see a `data` directory being created?

**Comment by Petrox** [comment]
Since the above mentioned situation I have created the /home/inventree/data and the issues instantly went away. 
(The data directory was not created during the docker compose command if I recall correctly).

I finally made it work, but I switched to inventree user, inventree home, etcetc.   I can live with this, just mentioned the error, since it was not mentioned in the docs.
